app/src/main/java/com/veniosg/dir/mvvm/model/FileHolder.java

```diff
@@ -140,6 +144,28 @@ public String getFormattedSize(Context c, boolean recursive){
 		return Formatter.formatFileSize(c, getSizeInBytes(recursive));
 	}
 
+	public String getNumberOfItems(Context context) {
+		if (context == null) {
```

Comment:
The error case check can be a single if, with inlined directory check. We also wouldn't need the empty string constant.

Reply:
@veniosg what should I return in that case then?

Reply:
Personally I'd even make context @Nonnull and only do `if(numOfDirectoryItems == 0) return "";` :)

Reply:
Sounds fine!